data/json/items/tools.json

```diff
@@ -123,6 +123,25 @@
     ],
     "magazine_well": 4
   },
+  {
+    "id": "UPS_off_air",
+    "type": "TOOL",
+    "name": "air fitting",
+    "name_plural": "air fittings",
+    "description": "This is an air fitting for a high pressure cylinder. It can connect a high pressure cylinder to a variety of devices utilizing air as power.",
+    "weight": 500,
+    "volume": "200 ml",
+    "price": 280000,
```

Comment:
$2800 for an air fitting?

Reply:
true

Reply:
That's a lot. Even if it's a RL-price, you should add a `price_postapoc` field with a much lower value as I doubt it will cost so much in a post-cataclysm world.